soh/soh/Enhancements/debugconsole.cpp

```diff
@@ -675,7 +637,7 @@ static bool NoZHandler(std::shared_ptr<Ship::Console> Console, const std::vector
     }
 
     try {
-        chaosEffectNoZ = std::stoi(args[1], nullptr, 10) == 0 ? 0 : 1;
+        GameInteractor_DisableZTargetingActive = std::stoi(args[1], nullptr, 10) == 0 ? 0 : 1;
```

Comment:
might be nice to just add `DisableZTargeting` and `EnableZTargeting` methods to gameinteractor instead of setting a var directly here